llvm/test/MC/Disassembler/AMDGPU/bf16_imm.txt

```diff
@@ -0,0 +1,8 @@
+# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1100 -disassemble -show-encoding < %s | FileCheck %s
+# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1200 -disassemble -show-encoding < %s | FileCheck %s
+
+# CHECK: v_dot2_bf16_bf16 v5, v1, v2, 0x42c8
```

Comment:
Add encoding to the check lines. Currently it is broken and encoded value is different from decoded one.